### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DarknesGaming**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ To-do
 ------------------------
 ###### Planned improvments
 * Internal Resolution changes
+* Synchronized Audio
```
Comment (Anuskuss) [inline README.md]: Ehm, we have synchronized Audio?

File: README.md [hunk 2]
```diff
@@ -26,20 +28,24 @@ Reporting Bugs
...
 ------------------------
 ###### Planned improvments
 * Internal Resolution changes
+* Synchronize Audio
 * GNU Lightning JIT (JAILBROKEN DEVICES ONLY)
 * Sharing roms between devices
 * Sharable game hacks
 * Add more localizations
+* DeSmuME 0.9.11 upgrade (minimum priority)
+* Power Optimizations
```
Comment (Anuskuss) [inline README.md]: What do you mean by that?

File: README.md (hunk 1 above)
Comment (DarknesGaming) [inline README.md]: It sounds terrible.

File: README.md (hunk 2 above)
Comment (DarknesGaming) [inline README.md]: Emulators tend to use a lot of power. iNDS is an emulator.

Comment (Anuskuss) [inline README.md]: Thanks for clarifying that. But my question is what can someone possibly do to optimize power consumtion and still keep a good framerate?.

Comment (Anuskuss) [inline README.md]: Thanks for clarifying that. But my question is what can someone possibly do to optimize power consumtion and still keep a good framerate?

Comment (DarknesGaming) [inline README.md]: Maybe @WilliamLCobb can add an option where you can choose to get maximum performance, maximum battery life, or balanced. I am not entirely sure.

**Pull request closed without merging**